docs/hooks/useTransactor.md

```diff
@@ -0,0 +1,72 @@
+---
+sidebar_position: 7
+---
+
+# useTransactor
+
+Use this hook to interact with the chain and get UI feedback on the transaction status. You can pass in anything that is a valid parameter to [Viem's `sendTransaction` function](https://viem.sh/docs/actions/wallet/sendTransaction#parameters).
```

Comment:
Probably we could  viem's `sendTransction`part frm here and just keep the first line? Since we also allow passing promise.

Reply:
https://github.com/scaffold-eth/se-2-docs/commit/4e965acb5c66f54fdca0692715e9a565804c9f8f